data_dir/scylla.repo.upgrade_2.0

```diff
@@ -0,0 +1,23 @@
+[scylla]
+name=Scylla for Centos $releasever - $basearch
+baseurl=http://downloads.scylladb.com/rpm/unstable/centos/branch-2.0/8/scylla/$releasever/$basearch/
```

Comment:
Can we use the `latest one` (http://downloads.scylladb.com/rpm/unstable/centos/branch-2.0/latest/)? then it's always the latest.